### PR TITLE
[1116023] Display event time in user's local time on event pages

### PR DESCRIPTION
**Pull request by Mte90**
Now:
screenshot4
After:
screenshot4

https://bugzilla.mozilla.org/show_bug.cgi?id=1116023

### CODE REVIEW

File: remo/base/static/base/css/app-fd4.less [hunk 1]
```diff
@@ -2471,13 +2471,15 @@ meter::-o-meter-suboptimum-value {
...
     .date-times {
         .opensanslight;
-        .grayed;
         text-align: left;
         border-left: 1px solid #ddd;
         border-bottom: none;
         text-align: left;
         min-width: 60px;
     }
+    .date-times::after {
```
Comment (akatsoulas) [inline remo/base/static/base/css/app-fd4.less]: Probably you miss typed one more `:`.

**Comment by akatsoulas** [comment]
Could you please also change the title of the commit to `[fix bug bug_number] text_here.`

**Comment by Mte90** [comment]
Merged and selector fixed, now works at 100%.
Is not easy contribute on the css part of this project.
Only in the folder /static/base/css the changes are used on the environment while on remo/static/base/css there are the changes to push.

**Comment by akatsoulas** [comment]
Thank you @Mte90  for this PR. r+ :rocket:

**Pull request merged**